src/Actor.h

```diff
@@ -499,14 +499,14 @@ class Actor : public MessageSubscriber
 	const TweenState& DestTweenState() const { return const_cast<Actor*>(this)->DestTweenState(); }
 
 	/** @brief How do we handle stretching the Actor? */
-	enum StretchType
+	enum class StretchType
 	{
 		fit_inside, /**< Have the Actor fit inside its parent, using the smaller zoom. */
```

Comment:
I'd also propose changing the naming to follow google style guide, which would be:
`kFitInside` and `kCover`